modules/backend/widgets/Form.php

```diff
@@ -452,6 +453,50 @@ public function onRefresh()
         return $result;
     }
 
+    /**
+     * Validates a single field in the context of the entire form contents.
+     *
+     * @return array `valid` (bool) whether the field input is valid
+     *               `message` (string) the validation error message, if any
+     */
+    public function onValidateField()
+    {
+        $saveData = $this->getSaveData();
+        $field = Input::get('fieldName');
```

Comment:
Typically we just use the `input()` or `post()` helper functions directly instead of the facade. (see https://github.com/octobercms/october/blob/master/modules/backend/widgets/Form.php#L1130)